modules/geolocation/geolocation.py

```diff
@@ -515,13 +530,24 @@ def run_locator(self, telemetry, coordinates):
         self.__cameraOrientation3v = camera_v
 
         point_pairs = self.gather_point_pairs()
+        # If insufficient point pairs, exit this run and try again
+        if len(point_pairs) < 4:
+            return False, None
+
         non_collinear_points = self.get_non_collinear_points(point_pairs)
+        # If insufficient point pairs, exit this run and try again
+        if len(non_collinear_points) < 4:
+            return False, None
 
         self.__pixelToGeoPairs = non_collinear_points
         tranformation_matrix = self.calculate_pixel_to_geo_mapping()
 
         geo_coordinates = self.map_location_from_pixel(tranformation_matrix, coordinates)
-        return geo_coordinates
+        return True, geo_coordinates
+
+    @staticmethod
+    def __convert_val_to_rad(convert_dict):
```

Comment:
Maybe deg_to_rad which is more explicit (can inputs be anything other than degrees?).